p3-weather-journal-app/website/app.js

```diff
@@ -3,3 +3,111 @@
 // Create a new date instance dynamically with JS
```

Comment:
function order is confusing

Reply:
done in: [0648763](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/7/commits/064876345af95540b541656b9526dcd75c2822f0)